includes/fbproduct.php

```diff
@@ -277,6 +277,33 @@ public function get_all_image_urls() {
 		return $image_urls;
 	}
 
+	/**
+	 * Gets a list of video URLs to use for this product in Facebook sync.
+	 *
+	 * @return array
+	 */
+	public function get_all_video_urls() {
+
+		$video_urls = array();
+
+		$attached_videos = get_attached_media( 'video', $this->id );
+		if ( empty( $attached_videos ) ) {
+			return $video_urls;
+		}
+		foreach ( $attached_videos as $video ) {
+			$url = $video->guid;
```

Comment:
```suggestion
			$url = wp_get_attachment_url( $video->ID );
```

`guid` is not reliable. Please use the WordPress function `wp_get_attachment_url` which will internally use guid as [a fallback](https://developer.wordpress.org/reference/functions/wp_get_attachment_url/#source).